x/mongo/driver/mongocrypt/options/mongocrypt_options.go

```diff
@@ -12,8 +12,10 @@ import (
 
 // MongoCryptOptions specifies options to configure a MongoCrypt instance.
 type MongoCryptOptions struct {
-	KmsProviders   bsoncore.Document
-	LocalSchemaMap map[string]bsoncore.Document
+	KmsProviders        bsoncore.Document
```

Comment:
Is `Kms` an acronym?  What is our policy on capitalizing acronyms (i.e. `KMSProviders`)?

Reply:
Yes. KMS stands for Key Management Service. There is no policy AFAIK. I see use of "Bson" and "BSON" as struct field names. https://github.com/golang/go/wiki/CodeReviewComments#initialisms suggests all caps for acronyms.

I prefer keeping as-is for the scope of this PR.